src/main/webapp/app/exercises/modeling/participate/modeling-submission.service.ts

```diff
@@ -67,11 +68,13 @@ export class ModelingSubmissionService {
      * @param {number} exerciseId - Id of the exercise
      * @param {boolean?} lock - True if assessment is locked
      */
-    getModelingSubmissionForExerciseWithoutAssessment(exerciseId: number, lock?: boolean): Observable<ModelingSubmission> {
-        let url = `api/exercises/${exerciseId}/modeling-submission-without-assessment`;
+    getModelingSubmissionForExerciseForCorrectionRoundWithoutAssessment(exerciseId: number, lock?: boolean, correctionRound?: number): Observable<ModelingSubmission> {
+        correctionRound = correctionRound ? correctionRound : 0;
```

Comment:
I am curious. I thinks this could be also shortened to `correctionRound = correctionRound ?? 0;`, or do i miss something?

Reply:
Thanks for the tip :D I didn't know this was possible

Reply:
It got mentioned in the meeting once, never had the chance to use it on my own, though. :D